cgi-bin/LJ/DB.pm

```diff
@@ -263,7 +263,7 @@ sub foreach_cluster {
     die $@ if $@;
 
     foreach my $cluster_id (@LJ::CLUSTERS) {
-        my $dbr = ($LJ::IS_DEV_SERVER) ?
+        my $dbr = %LJ::CLUSTER_PAIR_ACTIVE ?
```

Comment:
No, that's the problem - currently it is calling get_inactive_db in prod. We don't have CLUSTER_PAIR_ACTIVE defined, so that dies.